Gems/AWSCore/Code/Tools/ResourceMappingTool/controller/view_edit_controller.py

```diff
@@ -69,17 +69,13 @@ def _convert_and_write_to_json(self, config_file_name: str) -> bool:
         json_dict: Dict[str, any] = \
             json_utils.convert_resources_to_json_dict(self._proxy_model.get_resources(), self._config_file_json_source)
 
-        configuration: Configuration = self._configuration_manager.configuration
-        if json_dict.get(json_utils.RESOURCE_MAPPING_ACCOUNTID_JSON_KEY_NAME) == \
-                json_utils.RESOURCE_MAPPING_ACCOUNTID_TEMPLATE_VALUE:
-            json_dict[json_utils.RESOURCE_MAPPING_ACCOUNTID_JSON_KEY_NAME] = configuration.account_id
-
```

Comment:
I don't quite remember this, what will happen if you remove this? and why do you need this before? something like even there is no account id in it, we don't force to have it?

Reply:
Previously, if the value for account ID is "EMPTY" (string) and developer saves the resource mapping config file using the resource mapping tool, we replace that string with the user ID automatically. This behavior should be removed as well since account ID is optional and we don't actually want to add account ID to the file any more.